paypal/paypal-messages-android
# feat: session id and device id

## Description

- Add session ID and device ID to Logger and setGlobalAnalytics
- Add ignore script to build steps
- Update error / api tests for better coverage
- Fix some warnings
- Update test coverage
- Make setGlobalAnalytics static
- Use a UUID as the default session ID

## Testing instructions

- Add setGlobalAnalytics to demo
- Run demo
- Check logs to see that device ID and session ID are passed in
